devex/vscode-extension/assets/baseSolutionTemplateFolder/enc/enc.c

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <stdio.h>
+
+// Include the trusted [[project-name]] header that is generated
+// during the build. This file is generated by calling the
+// sdk tool oe_edger8r against the [[project-name]].edl file.
```

Comment:
No underscore, it's just oeedger8r.  Same in other files, so search for all occurrences of oe_edger8r

Reply:
Fixed.